NetworkChatClient/src/main/java/ru/gb/java2/kochemasov/controllers/ViewController.java

```diff
@@ -48,31 +47,36 @@ private void sendMessage() {
             return;
         }
 
-        String sender = null;
+        String destinator;
         if (!userList.getSelectionModel().isEmpty()) {
-            sender = userList.getSelectionModel().getSelectedItem();
-        }
+            destinator = userList.getSelectionModel().getSelectedItem();
+        } else destinator = "broadcast";
 
         try {
-            message = sender != null ? String.format("%s %s", sender, message) : message;
-            Network.getInstance().sendMessage(message);
+
+            message = String.format("/w %s %s %s", destinator, message, AuthController.getUsername());
+            if(!destinator.equals(AuthController.getUsername())) Network.getInstance().sendMessage(message);
         } catch (IOException e) {
             application.showNetworkDialog("Ошибка передачи данных по сети", "Не удалось отправить сообщение");
         }
-        appendMessageToChat("Me", message);
+        if(!destinator.equals(AuthController.getUsername())) appendMessageToChat("Me", message);
     }
 
     private void appendMessageToChat(String sender, String message) {
-        chatArea.appendText(DateFormat.getDateTimeInstance().format(new Date()));
-        chatArea.appendText(System.lineSeparator());
-        if (sender != null) {
-            chatArea.appendText(sender + ":");
+        String[] parsed = message.split(" ");
```

Comment:
```suggestion
        String[] parsed = message.split(" ", 3);
```
иначе сообщение с пробелами обработаем некорректно